x-pack/platform/plugins/private/data_visualizer/public/lite/file_picker.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { EuiFilePickerProps } from '@elastic/eui';
+import { EuiFormRow, EuiFilePicker, EuiSpacer } from '@elastic/eui';
+import type { EuiFilePickerClass } from '@elastic/eui/src/components/form/file_picker/file_picker';
+import { i18n } from '@kbn/i18n';
+import type { FC } from 'react';
+import React, { useCallback, useRef } from 'react';
+import type { FileManager } from './file_manager';
+
+interface Props {
+  fileManager: FileManager;
+}
+
+export const FilePicker: FC<Props> = ({ fileManager: fm }) => {
+  const filePickerRef = useRef<EuiFilePickerClass>(null);
+
+  const onFilePickerChange = useCallback(
+    (files: FileList | null) => {
+      if (files && files.length > 0) {
+        fm.addFiles(files).then((res) => {
+          filePickerRef.current?.removeFiles();
+        });
+      }
+    },
+    [fm]
+  );
+
+  return (
+    <>
+      <EuiFormRow
+        fullWidth
+        helpText={i18n.translate(
+          'xpack.dataVisualizer.file.aboutPanel.supportedFormatsDescription',
+          {
+            defaultMessage: 'Supported formats: PDF, TXT, MD, CSV, docx',
```

Comment:
Should we also mention supporting log files, NDJSON etc?

Reply:
Updated in https://github.com/elastic/kibana/pull/206864/commits/6f0ae208a785855a3c91c57fe9d31d515bd616a0